src/epgsql_sock.erl

```diff
@@ -126,30 +126,48 @@ handle_cast(cancel, State = #state{backend = {Pid, Key},
                              gen_tcp -> inet:peername(TimedOutSock);
                              ssl -> ssl:peername(TimedOutSock)
                          end,
-    SockOpts = [{active, false}, {packet, raw}, binary],
+    SockOpts = [{active, false}, {packet, raw}, binary,
+                {keepalive, true}],
```

Comment:
The latter. I can remove it in the interest of not changing more than necessary.